docs/testing.md

```diff
@@ -48,11 +48,53 @@ Or if you want to run bats from a different repository or branch, edit `centos7-
 
 Under `pipelines` are a series of playbooks designed around testing scenarios for various version of the Foreman and Katello stack. To run one:
 
-    ansible-playbook pipelines/pipeline_katello_nightly.yml -e "forklift_state=up"
+    ansible-playbook pipelines/<pipeline>.yml -e forklift_state=up -e <vars required by pipeline>
 
 When you are finished with the test, you can tear down the associated infrastructure:
 
-    ansible-playbook pipelines/pipeline_katello_nightly.yml -e "forklift_state=destroy"
+    ansible-playbook pipelines/<pipeline>.yml -e forklift_state=destroy -e <vars required by pipeline>
+
+### Existing Pipelines
+
+* `katello_pipeline` - Installs a Katello and a Content Proxy VMs and runs the `foreman_testing` role to verify the setup.
+  Expects the `katello_version` variable to be set to a known version (currently: 3.8, 3.9, 3.10, nightly)
+* `katello_upgrade_pipeline` - Installs a Katello VM, upgrades it twice and runs the `foreman_testing` role to verify the final upgrade.
+  Expects the `katello_version` variable to be set to a known version (currently: 3.8, 3.9, 3.10, nightly).
+* `pipeline_foreman_nightly` - Installs a nightly Foreman VM and runs the `foreman_testing` role to verify the setup.
+
+#### Examples
+
+    ansible-playbook pipelines/katello_upgrade_pipeline.yml -e forklift_state=up -e katello_version=nightly
+    ansible-playbook pipelines/katello_pipeline.yml -e forklift_state=up -e katello_version=3.10
```

Comment:
Can do in a follow up, I've been thinking about making `up` default. Thoughts?

Reply:
Sounds good, but doesn't belong in here ;)